index.js

```diff
@@ -123,25 +123,6 @@ class Encore {
         return this;
     }
 
-    /**
-     * Allows you to configure the options passed to the extract-text-webpack-plugin.
-     * A list of available options can be found at https://github.com/webpack-contrib/extract-text-webpack-plugin
-     *
-     * For example:
-     *
-     *      Encore.configureExtractTextPlugin((options) => {
-     *          options.ignoreOrder = true;
-     *      })
-     *
-     * @param {function} extractTextPluginOptionsCallback
-     * @returns {Encore}
-     */
-    configureExtractTextPlugin(extractTextPluginOptionsCallback = () => {}) {
```

Comment:
Maybe we could also do something in the `EncoreProxy` at the bottom of that file to detect when a user tries to call `configureExtractTextPlugin` or `enableCoffeeScriptLoader` and display some kind of "This method is not available anymore in Webpack Encore" error message instead of the default one?